README.md

```diff
@@ -8,28 +8,17 @@ these members to elastic search.
 ## Installation
 
 
-## :nut_and_bolt: Installation && Development
-The `run.sh` script lets you different options to run various part of the program:
-```
-chmod +x ./run.sh
-./run.sh
-```
-Please note that if you want to run the integration test(s) you need an elastic search
-cluster running locally. In that case, before running any test do:
-```
-docker-compose up
-```
-Afterwards, you can run the integration test.
-> **Note**: To shut down the cluster: docker-compose down
+## :nut_and_bolt: Development
+To run the unit test(s) do `run-unit-tests.sh`
```

Comment:
Paths to any scripts should use the relative path for the script to make it easier to find/copy and run. This works wonderfully from the terminal now though. 😃 

```suggestion
To run the unit test(s) do `scripts/run-unit-tests.sh`
```